src/main/java/com/aws/greengrass/util/Exec.java

```diff
@@ -275,7 +279,8 @@ public Exec withTimeout(long t, TimeUnit u) {
 
     @SuppressWarnings("PMD.SystemPrintln")
     public Exec withDumpOut() {
-        return withOut(l -> System.out.println("stderr: " + l)).withErr(l -> System.out.println("stdout: " + l));
+        return withOut(l -> System.out.println("stderr: " + l))
+                .withErr(l -> System.out.println("stdout: " + l));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are using `println()` rather than a dedicated logging facility makes it difficult to monitor the program behavior. We recommend to use a Java logging facility rather than `System.out` or `System.err`.

[Learn more](https://cwe.mitre.org/data/definitions/398.html)